Utilities/DataLoading/EdFi.XmlLookup.Console/Application/XmlLookupConfiguration.cs

```diff
@@ -155,18 +155,32 @@ public static XmlLookupConfiguration Create(IConfiguration configuration)
                 MaxSimultaneousRequests = configuration.GetValue("Concurrency:MaxSimultaneousApiRequests", 500),
                 ApiUrl = ResolvedUrl(configuration.GetValue<string>("OdsApi:ApiUrl")),
                 MetadataUrl = ResolvedUrl(configuration.GetValue<string>("OdsApi:MetadataUrl")),
+                XsdMetadataUrl = ResolvedUrl(configuration.GetValue<string>("OdsApi:XsdMetadataUrl")),
```

Comment:
Also define and initialize the InstanceId property (similar to what you did in BulkLoadClientConfiguration.cs).

Reply:
Code changed as requested.